send/crocs_user_export.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import sys
+import os.path
+import send_lib
+from datetime import datetime
+import re
+
+SERVICE_NAME = "crocs_user_export"
+
+TIMEOUT = "5400" #90s * 60 sec = 1.5h
+TIMEOUT_KILL = "60" # 60 sec to kill after timeout
+
+send_lib.check_input_fields(sys.argv)
+
+facility = sys.argv[1]
+destination = sys.argv[2]
+destination_type = send_lib.DESTINATION_TYPE_SERVICE_SPECIFIC if len(sys.argv) < 4 else sys.argv[3]
+
+service_files_dir = send_lib.get_gen_folder(facility, SERVICE_NAME)
+
+#unless specific configuration for destination exists use common configuration for all destination
+service_files_for_destination = service_files_dir + "/_destination/" + destination
```

Comment:
what is this variabe for?

Reply:
I have checked with @zlamalp and it is most likely some artifact of the past and can be removed. However we should check if the gen script does not try to generate files in that location.